hack/tools.mk

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This make file is supposed to be included in the top-level make file.
+# It can be reused by repos vendoring g/g to have some common make recipes for building and installing development
```

Comment:
Why don't you reuse the vendored `hack/tools.mk` file from g/g exactly like stated in this comment?

Reply:
Done